bolt-sidecar/src/config/mod.rs

```diff
@@ -180,8 +183,13 @@ impl TryFrom<Opts> for Config {
             config.limits.max_committed_gas_per_slot = max_committed_gas;
         }
 
-        config.commit_boost_url =
-            opts.signing.commit_boost_url.as_ref().map(|url| Url::parse(url)).transpose()?;
+        if let Some(commit_boost_url) = &opts.signing.commit_boost_url {
+            if let Ok(url) = Url::parse(commit_boost_url) {
+                if let Ok(socket_addrs) = url.socket_addrs(|| None) {
```

Comment:
what's with the `|| None` closure? just curious

Reply:
This is for matching the url schema, since we dont have any here so have to specify None. 

> If the URL has the default port number of a scheme that is unknown to this library, default_port_number provides an opportunity to provide the actual port number. In non-example code this should be implemented either simply as || None, or by matching on the URL’s .scheme().

Reply:
got it ty